homeassistant/components/insteon/binary_sensor.py

```diff
@@ -50,11 +52,18 @@
 }
 
 
-async def async_setup_platform(hass, config, async_add_entities, discovery_info=None):
-    """Set up the INSTEON entity class for the hass platform."""
-    async_add_insteon_entities(
-        hass, DOMAIN, InsteonBinarySensorEntity, async_add_entities, discovery_info
-    )
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the Insteon binary sensors from a config entry."""
+
+    def add_entities(discovery_info=None):
+        """Add the Insteon entities for the platform."""
+        async_add_insteon_entities(
+            hass, DOMAIN, InsteonBinarySensorEntity, async_add_entities, discovery_info
+        )
+
+    signal = f"{SIGNAL_ADD_ENTITIES}_{DOMAIN}"
```

Comment:
I was looking though `utils.py`, and we send

```
              signal = f"{SIGNAL_ADD_ENTITIES}_{platform}"
                dispatcher_send(hass, signal, {"address": device.address})
```


Should these be the platform instead of the domain below?

```
    signal = f"{SIGNAL_ADD_ENTITIES}_{DOMAIN}"
```

Reply:
I think the current code is right because it needs to be added to the specific platform and a domain qualifier to the signal is already included to separate it from other potential conflicts.  Just FYI, the `SIGNAL_ADD_ENTITIES` is currently defined as
```
SIGNAL_ADD_ENTITIES = "insteon_add_entities"
```
So the final signal is `insteon_add_entities_light` for example.

Reply:
It is the non-standard use of `DOMAIN` that is confusing.   Usually we use that to refer the the integration in all the integrations code.  All the reviewers are going to expect `DOMAIN` to be `insteon`

I'll add a suggested change as it will make it more clear as to what the concern is.